odoo/tools/__init__.py

```diff
@@ -4,7 +4,6 @@
 from . import pycompat
 from . import win32
 from . import appdirs
-from . import pdf
```

Comment:
For compatibility, this import should stay.

Reply:
I see, changing the way it was imported is most likely a mistake I made. Should I remove the other one I added `from .pdf import *` and keep this one but put it below `from .translate import *`?
I moved it below because I am using the translations _() and it was included later than the pdf.
(And changed the way it's written to fit with the other ones there, but that was probably a bad idea)